src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -57,4 +57,14 @@ public List<String> getAllValues(Prefix prefix) {
     public String getPreamble() {
         return getValue(new Prefix("")).orElse("");
     }
+
+    /**
+     * Validates whether multiple prefixes are stored in Argument Multimap.
+     *
+     * @return true if 2 or more prefixes are stored
+     */
+    public boolean containsMultiplePrefix() {
+        Integer sizeOffSet = argMultimap.size() - 1;
```

Comment:
Nope, it is meant to be offSet because the multimap contains an extra object in the map other than the prefixes, so I have to minus one to offset it to the correct size.

Reply:
Should the naming be `sizeOffset`?